test/over_react_redux/fixtures/connect_flux_counter.dart

```diff
@@ -53,18 +53,18 @@ class ConnectFluxCounterComponent
         ..addTestId('button-increment')
         ..onClick = (_) {
           if (props.mutateStoreDirectly != null) {
-            props.mutateStoreDirectly();
+            props.mutateStoreDirectly!();
           } else if (props.increment != null) {
-            props.increment();
+            props.increment!();
           } else {
-            props.actions.incrementAction();
+            props.actions!.incrementAction(1);
```

Comment:
Is the addition of this argument intentional?

Reply:
Looks like that was leftover from when I was using a different w_flux null-safety branch; reverted: 5b56c14d512997e83b83c16de66654184d3b2ce2